teuthology/task/internal.py

```diff
@@ -272,10 +272,13 @@ def add_remotes(ctx, config):
     """
     remotes = []
     machs = []
+    user = teuth_config.test_user
+    if not user:
+        user = 'ubuntu'
```

Comment:
If you have a default set properly as I mentioned in my other comment, you won't need this block.